.NET/Microsoft.Recognizers.Text.DateTime/Parsers/CJK/BaseCJKDateTimePeriodParser.cs

```diff
@@ -0,0 +1,598 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using Microsoft.Recognizers.Text.DateTime.Utilities;
+using Microsoft.Recognizers.Text.Utilities;
+using DateObject = System.DateTime;
+
+namespace Microsoft.Recognizers.Text.DateTime
+{
+    public class BaseCJKDateTimePeriodParser : IDateTimeParser
+    {
+        public static readonly string ParserName = Constants.SYS_DATETIME_DATETIMEPERIOD;
+
+        private readonly ICJKDateTimePeriodParserConfiguration config;
+
+        public BaseCJKDateTimePeriodParser(ICJKDateTimePeriodParserConfiguration configuration)
+        {
+            config = configuration;
+        }
+
+        public static string BuildTimex(TimeResult timeResult)
```

Comment:
This should move to TimexUtil and merge with something equivalent already there.

Reply:
it turned out this method was not used anywhere, so I deleted it.